script.user.js

```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name         Flyff Universe Multi-Tab Spell Caster
-// @namespace    pPrecel
-// @version      2.0
+// @namespace    pPrecel, doodlefudge
```

Comment:
should the namespace be comma separated? 🤔 I guess a list of authors should be reserved for the `@author` field